.gitmodules

```diff
@@ -22,3 +22,6 @@
 [submodule "external/debugger"]
 	path = external/debugger
 	url = https://github.com/kevinb7/debugger
+[submodule "external/p5"]
+	path = external/p5
+	url = https://github.com/kevinb7/p5.js
```

Comment:
It worries me a bit to move to P5.JS without doing a lot of testing, and without having an official team to tackle issues (I'm currently the only eng on CS, and I'm creating content this week). One thing I could do is deploy this change with p5.JS to a non-default server, and have you poke at a bunch of programs and watch talk-throughs.